BUILD.gn

```diff
@@ -313,11 +313,15 @@ if (current_toolchain != "${dir_pw_toolchain}/default:default") {
     enable_genio_builds = false
   }
 
+  # Pigweed does not provide a clang in some configurations.
+  if (enable_default_builds) {
+    _have_pigweed_clang =
+        host_os != "win" && (host_os != "mac" || host_cpu != "arm64")
```

Comment:
I understand this was copied from previous code but this might look more intuitive: `!(host_os == "mac" && host_cpu == "arm64")`

Not required to merge - feel free to close this conversation and merge if you don't feel this matters.

Reply:
You're right, that's probably more readable.  Will update.